assets/src/application/ui/calendars/dateDisplay/TimezoneTimeInfo/index.tsx

```diff
@@ -33,14 +42,19 @@ const TimezoneTimeInfo: React.FC<OffsetInfoProps> = ({ className, date, inline =
 			<div className={'ee-focus-priority-6'}>{formatUtcDateForSite(date)}</div>
 		</div>
 	);
-	const htmlClassName = classNames(className, 'ee-timezone-info', { 'ee-timezone-info--inline': inline });
+	const label = __('click for timezone\ninformation');
+
 	return (
-		<Popover content={content} title={__('This Date Converted To:')} trigger={'click'} align={{ offset: [0, -60] }}>
-			<div className={htmlClassName}>
-				<Tooltip title={__('click for timezone\ninformation')} mouseEnterDelay={1}>
-					<GlobalOutlined />
-				</Tooltip>
-			</div>
+		<Popover>
+			<PopoverTrigger>
+				<IconButton aria-label={label} icon={GlobalOutlined} variant='unstyled' />
```

Comment:
tried wrapping this with a tooltip, but got an error:
![image](https://user-images.githubusercontent.com/1477453/78836719-564f2100-79fb-11ea-8204-1ccdae777da7.png)
***
seems like `PopoverTrigger` children need to be either `Button` or `IconButton` or smth that can use `forwardRef`, tried `EspressoButton` but it didn't worked